firebase.json

```diff
@@ -5,8 +5,8 @@
   },
   "functions": {
     "predeploy": [
-      "npm --prefix \"$RESOURCE_DIR\" run lint",
-      "npm --prefix \"$RESOURCE_DIR\" run build"
+      "npm --prefix ./functions run lint",
+      "npm --prefix ./functions run build"
```

Comment:
https://github.com/firebase/firebase-tools/issues/610#issuecomment-360147507
`\"$RESOURCE_DIR\"` fails on Windows, but `\"%RESOURCE_DIR%\"` fails on Mac and Linux. To get deployment working on all the platforms, the easiest way is to hard-code the `functions` directory name.